presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -334,44 +339,54 @@ public void testRenameTable()
     }
 
     @Test
-    public void testCommentTable()
+    public final void testCommentTable()
```

Comment:
I marked this final on purpose, to help implementors of connectors outside of the repository to update the code.
Previous test suppression model was via test override. This is no longer the way, and the new test version also verifies the connector is correctly reporting lack of support for comments.

WDYT?

Reply:
The testDelete case shows this can be more subtle.

Reply:
> Allow reading ORC files which do not have row-group information

Why?

Reply:
because supporting a feature is non-binary -- hive supports delertes, but only certain form of them